google/cloud/pubsub/samples/samples.cc

```diff
@@ -1071,22 +1071,29 @@ void CustomThreadPoolSubscriber(std::vector<std::string> const& argv) {
     auto subscription =
         pubsub::Subscription(std::move(project_id), std::move(subscription_id));
 
+    // Because this is an example we want to exit eventually, use a mutex and
+    // condition variable to notify the current thread and stop the example.
     std::mutex mu;
+    std::condition_variable cv;
```

Comment:
Nit: Because it is an example, we should probably also explicitly include `<mutex>` and `<condition_variable>`.

Reply:
Done.